Makefile

```diff
@@ -36,15 +38,19 @@ ifdef NEON64_CFLAGS
 endif
 ifdef SSSE3_CFLAGS
   HAVE_SSSE3 = 1
+  HAVE_FAST_UNALIGNED_ACCESS = 1
 endif
 ifdef SSE41_CFLAGS
   HAVE_SSE41 = 1
+  HAVE_FAST_UNALIGNED_ACCESS = 1
 endif
 ifdef SSE42_CFLAGS
   HAVE_SSE42 = 1
+  HAVE_FAST_UNALIGNED_ACCESS = 1
```

Comment:
It might not be useless when implementing white character filtering. So I Iet the whole process in place. I could remove that if necessary.